### PR TITLE
[platform_tests/api]: Add watchdog config for Celestica Seastone device

### PR DESCRIPTION
**Pull request by mudsut4ke**
### Description of PR

Summary: Add watchdog config for Celestica Seastone device

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

#### What is the motivation for this PR?
- To fixes watchdog failed test case on Celestica Seastone platform api testbed

#### How did you do it?
1. Add the definiation of too_big_timeout value in watchdog.yml which will be used to compare to the seastone watchdog max time setting.

2. Add the too_big_timeout checking in watchdog.py of seastone DUT platform definiation.
If the too_big_timeout value is bigger than the watchdog max time setting value, test case will return Error which is expected test result.

#### How did you verify/test it?
- Through running Testbed pytest platform watchdog test case.

#### Any platform specific information?
- Celestica seastone platform

#### Supported testbed topology if it's a new test case?
- Yes

### CODE REVIEW

File: tests/platform_tests/api/watchdog.yml [hunk 1]
```diff
@@ -62,3 +62,10 @@ x86_64-arista.*:
     valid_timeout: 10
     greater_timeout: 100
     too_big_timeout: 660
+
+# Celestica Seastone watchdog
+x86_64-cel_seastone-r0:
+  default:
+    valid_timeout: 10
```
Comment (wangxin) [inline tests/platform_tests/api/watchdog.yml:69]: The `valid_timeout` is same as the default value. It is better to remove it here. Then the default value will be taken.

Comment (mudsut4ke) [reply tests/platform_tests/api/watchdog.yml]: updated https://github.com/Azure/sonic-mgmt/pull/3172/commits/b8aa1db282aac3d7082889f8c8eaaafc3686881b

**Review by wangxin — APPROVED** [review]

**Review by jleveque — APPROVED** [review]

**Pull request merged**